src/test/java/com/google/api/codegen/testsrc/common/common_resources.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains stub messages for common resources in GCP.
+// It is not intended to be directly generated, and is instead used by
+// other tooling to be able to match common resource patterns.
+syntax = "proto3";
+
+package google.cloud;
```

Comment:
This file is about to change to use the new file-level annotation (rather than using proto messages).
Should this PR be updated to this new format?

Reply:
Could be the right thing to do. It will delay this PR and resource-name plugin one, because same thing will need to be implemented in two places, probably in quite different way, because both gapic-generator and resource name plugin try to do same thing but in a different way =(.

Reply:
Please submit this PR as-is and let's make a new one to add support for the file level annotation.

I will not change `common_resources.proto` until gapic-generator supports it.